server/app/repository/AccountRepository.java

```diff
@@ -469,6 +474,13 @@ public void addIdTokenAndPrune(AccountModel account, String sessionId, String id
     }
     idTokens.purgeExpiredIdTokens(clock);
     idTokens.storeIdToken(sessionId, idToken);
+
+    if (settingsManifest.getSessionReplayProtectionEnabled()) {
+      // For now, we set the duration to Auth0s default of 10 hours.
+      account.removeExpiredActiveSessions(clock, Duration.ofHours(10));
```

Comment:
(thought) It might be more work, but rather than pushing the clock and the duration as separate things a new class tailored to the session lifecycle would go a long way to making the intent and tests easier to read.

```psueudocode
class SessionLifecycle {
  ctor(Clock) {}
  Duration maxSessionDuration() {}
  ...
}
```

Reply:
Updated to create that class! And also agree around session duration being configurable - created a bug to track and added a TODO